### ISSUE TITLE
Error "attempt to index field 'THNN' (a nil value)"

### ISSUE THREAD

**Issue by Sugarbank** [junyanz/CycleGAN#18]
------------------- Options -------------------	
                DATA_ROOT: ./datasets/ae_photos	
               align_data: 0	
             aspect_ratio: 1	
                batchSize: 1	
          checkpoints_dir: ./checkpoints	
           continue_train: 1	
                    cudnn: 1	
                  display: 1	
               display_id: 200	
                 fineSize: 256	
                     flip: 0	
                      gpu: 1	
                 how_many: all	
                 input_nc: 3	
                 loadSize: 256	
                    model: one_direction_test	
                 nThreads: 1	
                     name: style_vangogh_pretrained	
                     norm: instance	
                output_nc: 3	
                    phase: test	
               preprocess: regular	
           resize_or_crop: scale_width	
              results_dir: ./results/	
           serial_batches: 1	
                     test: 1	
          which_direction: AtoB	
              which_epoch: latest	
-----------------------------------------------	
GPU Mode	
function: 0x4047acd8	
DataLoader UnalignedDataLoader was created.	
Starting donkey with id: 1 seed: 8091
table: 0x41e940c0
table: 0x418ef650
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..wc -L '/tmp/lua_gOkqLo' |cut -f1 -d' '
205 samples found................................................................... 0/205 ....................................................................................]ETA: 0ms | Step: 0ms           
Updating classList and imageClass appropriately
 [================================================================================== 1/1 =====================================================================================>]ETA: 0ms | Step: 0ms           
Cleaning up temporary files
Dataset Size A: 	205	
Starting donkey with id: 1 seed: 2932
table: 0x414e6458
table: 0x414a2c68
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..wc -L '/tmp/lua_WJyu1k' |cut -f1 -d' '
205 samples found................................................................... 0/205 ....................................................................................]ETA: 0ms | Step: 0ms           
Updating classList and imageClass appropriately
 [================================================================================== 1/1 =====================================================================================>]ETA: 0ms | Step: 0ms           
Cleaning up temporary files
Dataset Size B: 	205	
use InstanceNormalization	
loading previously trained model (/home/ubuntu/CycleGAN/checkpoints/style_vangogh_pretrained/latest_net_G.t7)	
use InstanceNormalization	
---------- # Learnable Parameters --------------	
G_A = 2855811	
------------------------------------------------	
processing batch 1	
pathsA	{
  1 : "20.jpg"
}
pathsB	nil	
/home/ubuntu/torch/install/bin/luajit: /home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:67: 
In 1 module of nn.Sequential:
...ch/install/share/lua/5.1/nn/SpatialReflectionPadding.lua:14: attempt to index field 'THNN' (a nil value)
stack traceback:
	...ch/install/share/lua/5.1/nn/SpatialReflectionPadding.lua:14: in function <...ch/install/share/lua/5.1/nn/SpatialReflectionPadding.lua:12>
	[C]: in function 'xpcall'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:63: in function 'rethrowErrors'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
	./models/one_direction_test_model.lua:44: in function 'Forward'
	test.lua:101: in main chunk
	[C]: in function 'dofile'
	...untu/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:131: in main chunk
	[C]: at 0x00406670

WARNING: If you see a stack trace below, it doesn't point to the place where this error occurred. Please use only the one above.
stack traceback:
	[C]: in function 'error'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:67: in function 'rethrowErrors'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
	./models/one_direction_test_model.lua:44: in function 'Forward'
	test.lua:101: in main chunk
	[C]: in function 'dofile'
	...untu/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:131: in main chunk
	[C]: at 0x00406670
test_vangogh_style_on_ae_photos.sh: 17: [: 1: unexpected operator

**Comment by Sugarbank** [comment]
for some reason I don't get these errors when running in cpu mode.
Have cuda and cudn etc installed for torch

**Comment by junyanz** [comment]
Have you tried the suggestions discussed in issue #17?
